lib/command/definitions.js

```diff
@@ -2,387 +2,160 @@ const getConfig = require('./utils').getConfig;
-  _test(test: () => void): void
-  /** Hook executed after each passed test */
-  _passed(test: () => void): void
-  /** Hook executed after each failed test */
-  _failed(test: () => void): void
-  /** Hook executed before each step */
-  _beforeStep(step: () => void): void
-  /** Hook executed after each step */
-  _afterStep(step: () => void): void
-  /** Hook executed before each suite */
-  _beforeSuite(suite: () => void): void
-  /** Hook executed after each suite */
-  _afterSuite(suite: () => void): void
-  /** Hook executed after all tests are executed */
-  _finishTest(suite: () => void): void
-  /**Access another configured helper: this.helpers['AnotherHelper'] */
-  readonly helpers: any
-  /** Print debug message to console (outputs only in debug mode) */
-  debug(msg: string): void
-
-  debugSection(section: string, msg: string): void
-}
-
-declare class Locator {
-  constructor(locator: LocatorOrString, defaultType?: string);
-
-  or(locator: LocatorOrString): Locator;
-  find(locator: LocatorOrString): Locator;
-  withChild(locator: LocatorOrString): Locator;
-  withDescendant(locator: LocatorOrString): Locator;
-  at(position: number): Locator;
-  first(): Locator;
-  last(): Locator;
-  inside(locator: LocatorOrString): Locator;
-  before(locator: LocatorOrString): Locator;
-  after(locator: LocatorOrString): Locator;
-  withText(text: string): Locator;
-  withAttr(attrs: object): Locator;
-  as(output: string): Locator;
 }
 
-
-declare function actor(customSteps?: {
-  [action: string]: (this: CodeceptJS.I, ...args: any[]) => void
-}): CodeceptJS.{{I}};
-declare function actor(customSteps?: {}): CodeceptJS.{{I}};
-declare function Feature(title: string, opts?: {}): FeatureConfig;
-declare const Scenario: {
-  (title: string, callback: ICodeceptCallback): ScenarioConfig;
-  (title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-  only(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  only(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-}
-declare interface IScenario {
-  Scenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  Scenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-}
-declare function xScenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-declare function xScenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-declare interface IData {
-  Scenario(title: string, callback: ICodeceptCallback): ScenarioConfig;
-  Scenario(title: string, opts: {}, callback: ICodeceptCallback): ScenarioConfig;
-  only: IScenario;
-}
+declare function actor<T extends { [action: string]: Function }>(
+  customSteps: T & ThisType<WithTranslation<methods & T>>
+): WithTranslation<methods & T>;
+declare function inject(): SupportObject;
+declare function inject<T extends keyof SupportObject>(name: T): SupportObject[T];
+declare const codecept_actor: typeof actor;
+declare const Scenario: Scenario;
```

Comment:
Cannot find name 'Feature'
![image](https://user-images.githubusercontent.com/11293201/62792921-ff462480-bad8-11e9-8c84-4c36b3d86073.png)

Reply:
This should have been a caching issue, make sure you reload the definition files before testing